src/test/e2e/cypress/e2e/search.cy.js

```diff
@@ -81,3 +81,12 @@ it('should rename files to subject-folder they are inside of after downloading',
     }
     cy.expect(i).to.equal(3)
 });
+
+it('should disable button if not enough numbers are in input', () => {
+    cy.get('input[type="text"]').type('110');
+    cy.get('button').should('not.be.disabled')
+});
+
+it('should disable button if input is empty', () => {
+    cy.get('button').should('be.disabled')
```

Comment:
Maybe extend test to test with whitespaces once `.trim()` is added

Reply:
added white space to test